src/main/java/com/kadai10/employee/controller/UserControllerAdvice.java

```diff
@@ -31,8 +34,45 @@ public ResponseEntity<Map<String, String>> handleUserNotFoundException(
                 "timestamp", ZonedDateTime.now().toString(),
                 "status", String.valueOf(HttpStatus.NOT_FOUND.value()),
                 "error", HttpStatus.NOT_FOUND.getReasonPhrase(),
-                "message", e.getLocalizedMessage(),
+                "message", "ユーザーが見つかりません",
                 "path", request.getRequestURI());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    /**
+     * 入力が不足している場合の例外ハンドリングメソッド。
+     * @param e       入力が不足している例外
+     * @param request HTTPリクエスト
+     * @return エラーレスポンス
+     */
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
```

Comment:
メソッドとJavadocの間に空行はない方がまとまりがわかって読みやすいです。
```suggestion
     */
    @ExceptionHandler(MethodArgumentNotValidException.class)
```

Reply:
アドバイスありがとうございます。
060fb34b21b078f7d683f32bd636906c1c7bf9e1 
メソッドとJavadocの間に空行を消しました。
ご確認お願いいたします。

Reply:
問題ないです。